interfaces/ingress/v0/charms.yaml

```diff
@@ -1,3 +1,9 @@
 providers:
-  - traefik-k8s
-consumers: []
+  - name: traefik-k8s
+    url: https://github.com/canonical/traefik-k8s-operator
+
+consumers:
```

Comment:
Maybe don't combine the refactor and adding new stuff to make the review easier.